Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -626,6 +630,39 @@ class CallController: @unchecked Sendable {
         }
     }
 
+    private func subscribeToCurrentUserBlockedState(_ call: Call?) {
+        disposableBag.remove("current-user-blocked")
```

Comment:
why do we need this?

Reply:
This is to remove the cancellable for the specific observation.